App/src/main/res/values/strings.xml

```diff
@@ -16,6 +16,7 @@
     <string name="settings">Settings</string>
     <string name="delete">Delete</string>
     <string name="rename">Rename</string>
+    <string name="logout">Log Out</string>
```

Comment:
I would consider using "Sign out" rather than "Log out" since the app is using "Sign in" rather than "Log in" as the name of the opposite action.